components/meta-packages/auto-install/auto_install.p5m

```diff
@@ -184,6 +184,7 @@ depend type=require fmri=network/ftp@0.5.11-0.151.1.8
 depend type=require fmri=network/ipfilter@0.5.11-0.151.1.8
 depend type=require fmri=network/iscsi/initiator@0.5.11-0.151.1.8
 depend type=require fmri=network/iscsi/iser@0.5.11-0.151.1.8
+depend type=require fmri=network/openssh@7.2.0.2-2015.0.2.0
```

Comment:
Hm, I do not understand how the version numbers listed here will help with your scenario (probably lack of pkg knowledge at my side).

Anyway, almost every version number in this file is already obsolete...